src/utils.js

```diff
@@ -221,4 +221,7 @@ global.utils = {
     return returnValue;
   },
 
+  lpadround: function lpadround(nr, lpad, digest) {
```

Comment:
I recently discovered `padStart` and `padEnd` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart)

Wondering if this wouldn't be easier to read. So something like `return nr.padStart(digest+1pad+1).padEnd(digest+1pad+2)` (Not sure if this matches exactly what you want to accomplish).

Reply:
It's a round function with leading spaces and fixed digest count.

Reply:
Ah, missed somehow the rounding, sry.

How about: `return nr.toFixed(digest).padStart(lpad + digest);`

Reply:
I changed it to your suggestion it's easier understandable.